packages/docs/src/components/Api/Comment/index.tsx

```diff
@@ -33,16 +33,7 @@ function Experimental({comment}: {comment: JSONOutput.Comment}) {
     [comment],
   );
 
-  return (
-    <>
-      {experimental && (
-        <Admonition type="experimental">
-          This is an experimental feature. The behavior and API may change
-          drastically between major releases.
-        </Admonition>
-      )}
-    </>
-  );
+  return <>{experimental && <ExperimentalWarning />}</>;
```

Comment:
(Nit) From what I understand of JSX, it's good practice to write this as:
```
{experimental ? <ExperimentalWarning /> : null}
```
This prevents `experimental` leaking into the DOM in certain conditions.

Reply:
Changed it and removed the wrapping fragment altogether.